lib/services/northBound/northboundServer.js

```diff
@@ -56,8 +56,8 @@ function start(config, callback) {
     northboundServer.app.set('port', config.server.port);
     northboundServer.app.set('host', config.server.host || '0.0.0.0');
     northboundServer.app.use(domainUtils.requestDomain);
-    northboundServer.app.use(bodyParser.json());
-    northboundServer.app.use(bodyParser.json({ type: 'application/*+json' }));
+    northboundServer.app.use(bodyParser.json({ limit: '1mb' }));
+    northboundServer.app.use(bodyParser.json({ limit: '1mb', type: 'application/*+json' }));
```

Comment:
Maybe this should be unhardwired to a configuration parameter?

Reply:
Is there any other similiar configuration parameter to do it in a similar way?